wdpk/zerotier/build.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/sh
-
+set -e
 APP_NAME="$(basename $(pwd))"
-DATE="$(date +"%m%d%Y")"
-CWD="$(pwd)"
 VERSION="$(awk '/Version/{print $NF}' apkg.rc)"
 
 echo "Building ${APP_NAME} version ${VERSION}"
 
 MODELS="MyCloudEX2Ultra"
 
 for model in $MODELS; do
```

Comment:
For the CI, I moved away from build scripts inside the package directories.
Most of these packages had the same stuff anyway.
It's still really ugly code, but you can find it [here](https://github.com/WDCommunity/wdpksrc/blob/master/.github/workflows/wdpk.yml)

Reply:
Do you prefer me to try to adapt that? I can check if inside the build dir there is a Makefile and run it